### PR TITLE
fix: Make babel-plugin-minify-infinity work again. (#839)

### PR DESCRIPTION
**Pull request by Cyp**
Here's an attempt at fixing #839. I don't know whether there are any other cases that need catching.

### CODE REVIEW

File: packages/babel-plugin-minify-infinity/src/index.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ module.exports = function({ types: t }) {
     t.numericLiteral(1),
     t.numericLiteral(0)
   );
+  const badTransforms = {
+    ArrayPattern: true,
+    AssignmentExpression: true,
```
Comment (boopathi) [inline packages/babel-plugin-minify-infinity/src/index.js]: Doesn't this prevent transforming `x = Infinity`?

**Comment by boopathi** [comment]
Ah!. I worked on the same thing and have #841 .

File: packages/babel-plugin-minify-infinity/src/index.js (hunk 1 above)
Comment (Cyp) [reply packages/babel-plugin-minify-infinity/src/index.js]: Yes, I missed testing that case.

**Comment by Cyp** [comment]
Don't know if it's the right approach, but I've updated my version in case.

**Comment by vigneshshanmugam** [comment]
Thanks for the PR @Cyp

Fixed by #841

**Pull request closed without merging**